proposals/wac-ucr/index.bs

```diff
@@ -1111,16 +1532,259 @@ specially generated link to access the document with [=read access=]. He sets
 it to [[#conditional-time|expire]] in three hours. Bob gives the link to Anne
 and the presentation goes off perfectly.
 
-Requirements {#requirements}
+* [[#req-vc]]
+* [[#req-read]]
+* [[#req-conditional-time]]
+
+Functional Requirements {#requirements}
 ================================================================================
 
-Issue: Populate requirements based on established use cases.
+Note: Unless otherwise specified, *allowing or limiting access* in the 
+subsequent requirements should be interpreted in the context of allowing
+or limiting access to a [=resource=] or [=collection=] by an [=agent=].
+
+## Access by subject ## {#req-as}
+
+### The system shall allow access to be limited based on the [=identity=] of the [=agent=]. ### {#req-agent-identity}
+
+<div class="assertion">    
+  Related use cases: [[#uc-basic]], [[#uc-collections]]
+</div>
+
+### The system shall allow access to be limited based on the [=identity=] of the [=agent=], only when that identity is issued by a trusted identity provider. ### {#req-trusted-identity}
+
+<div class="assertion">    
+  Related use cases: [[#uc-trustedissuers]]
+</div>
+
+### The system shall allow access to be limited to an [=agent=] based on the [=agent's=] membership in a certain group of [=agents=]. ### {#req-agent-group}
+
+<div class="assertion">    
+  Related use cases: [[#basic-group]]
+</div>
+
+### The system shall allow access to be limited to an [=agent=] based on the client [=application=] in use by the [=agent=]. ### {#req-application}
+
+<div class="assertion">    
+  Related use cases: [[#uc-trusted-applications]]
+</div>
+
+### The system shall allow an [=agent=] to limit modes and/or conditions of access for a given client [=application=] in their use for a [=resource=] or [=collection=] that they have been granted access to. ### {#req-client-constrained}
+
+<div class="assertion">    
+  Related use cases: [[#uc-client-constraints]]
+</div>
+
+### The system shall allow access to be permitted for any unauthenticated or authenticated [=agent=]. ### {#req-public}
+
+<div class="assertion">    
+  Related use cases: [[#basic-public]]
+</div>
+    
+### The system shall allow access to be limited to any authenticated [=agent=]. ### {#req-authenticated}
+    
+<div class="assertion">    
+  Related use cases: [[#basic-authenticated]]
+</div>
+
+## Access by capability ## {#req-capabilities}
+
+### The system shall allow access to be limited to an [=agent=] based on the [=agent's=] possession of a certain verifiable credential or capability. ### {#req-vc}
+
+<div class="assertion">    
+  Related use cases: [[#capabilities-vc]], [[#capabilities-link]], [[#uc-minimalcredentials]]
+</div>
+
+## Access to resources ## {#req-resources}
+
+### The system shall limit the ability to read the access permissions associated with a certain [=resource=]. ### {#req-read-permissions}
+
```

Comment:
Above we had a "shall allow access to be limited to..." 
Here we have "The system shall limit the ability to read..." 
Clearly if we want the use cases for payment or innumerable use cases where access control rules are open, then this should be up to the owner of the resource to decide if the rules associated with it are visible or not. If they are not visible then how can the client know how to connect?

Reply:
Adjusted in https://github.com/solid/authorization-panel/pull/152/commits/691e677fb30249fd00b4d3a547899a98a7713f51